src/test/java/com/amazon/dlic/auth/http/saml/HTTPSamlAuthenticatorTest.java

```diff
@@ -171,7 +171,7 @@ public void basicTest() throws Exception {
             .put(IDP_METADATA_URL, mockSamlIdpServer.getMetadataUri())
             .put("kibana_url", "http://wherever")
             .put("idp.entity_id", mockSamlIdpServer.getIdpEntityId())
-            .put("exchange_key", "abc")
+            .put("exchange_key", "6aff3042-1327-4f3d-82f0-40a157ac4464")
```

Comment:
Can there be 2 separate tests to confirm key behavior in both scenarios?

Reply:
done